src/main/java/com/jfrog/ide/common/configuration/JfrogCliDriver.java

```diff
@@ -73,15 +80,23 @@ public JfrogCliServerConfig getServerConfig(File workingDirectory, List<String>
         }
     }
 
-    public String version(File workingDirectory) throws IOException, InterruptedException {
-        return runCommand(workingDirectory, new String[]{"--version"}, Collections.emptyList()).getRes();
+    public String version(File workingDirectory) {
```

Comment:
If you are already changing this, please try to use `Version` object from build-info instead of string